axlearn/cloud/gcp/jobs/bastion_vm.py

```diff
@@ -249,6 +249,9 @@ def _gcloud_storage_rsync(
             timeout=timeout_s,
             capture_output=True,
             text=True,
+            # Avoid "No space left on device":
+            # https://cloud.google.com/knowledge/kb/error-message-while-running-the-command-gsutil-rsync-000004577
+            env={"TMPDIR": f"{_LOG_DIR}/rsync"},
```

Comment:
nit -- I wonder if we should use /var/tmp/rsync explicitly, given that log dir is often the directory being rsync'ed itself?

Reply:
Good point. Done.